src/tool_listhelp.c

```diff
@@ -288,16 +288,16 @@ const struct helptxt helptext[] = {
    "Use HTTP 1.1",
    CURLHELP_HTTP},
   {"    --http2",
-   "Use HTTP/2",
+   "Use HTTP 2",
    CURLHELP_HTTP},
   {"    --http2-prior-knowledge",
    "Use HTTP 2 without HTTP/1.1 Upgrade",
```

Comment:
```suggestion
   "Use HTTP 2 without HTTP 1.1 Upgrade",
```
Also this to stay consistent?

Or the reverse? The repo overall is now siding toward the slash-style. This is also what used more in general, at least in my experience.

Reply:
Hm, yes, I'll try using the slash-style...

Reply:
I like it:

~~~
$ curl -h all | grep -- --http
     --http0.9                                     Allow HTTP/0.9 responses
 -0, --http1.0                                     Use HTTP/1.0
     --http1.1                                     Use HTTP/1.1
     --http2                                       Use HTTP/2
     --http2-prior-knowledge                       Use HTTP/2 without HTTP/1.1 Upgrade
     --http3                                       Use HTTP/3
     --http3-only                                  Use HTTP/3 only